util/test/tempus.go

```diff
@@ -11,6 +11,7 @@ import (
 	"time"
 )
 
+// FIXME: Abort long running tests
```

Comment:
Anything you plan to do in this PR about this?

Reply:
No. This was a thought that struck me when passing by. I'll create a ticket for tracking.